packages/v4/src/content/design-guidelines/components/text-input/text-input.md

```diff
@@ -47,5 +47,5 @@ Wherever possible, use inline errors to notify users where there is an issue wit
 
 <img src="./img/error.png" alt="error and validation example" width="320"/>
 
-For more information about how to provide errors and validation, see  the [forms design guidelines](/design-guidelines/usage-and-behavior/forms#errors-and-validation).
+For more information about how to provide errors and validation, see  the [forms design guidelines](/components/forms/design-guidelines#errors-and-validation).
```

Comment:
Same here: The link should be /components/form/design-guidelines.... etc,  with "form" being singular, not plural